Makefile

```diff
@@ -423,6 +385,17 @@ ifdef CODE_SIGN_CERTIFICATE_PASSWORD
 endif
 endif
 
+assets/osx/installer.tiff: assets/osx/installer.png assets/osx/installer@2x.png
+	tiffutil -cathidpicheck $^ -out $@
```

Comment:
If it's _always_ built-in, then it's probably fine (much like we don't document the need for `cp` or `mv`).